packages/web/components/alert/toast.tsx

```diff
@@ -1,11 +1,15 @@
 import Image from "next/image";
-import { FunctionComponent } from "react";
+import React, { FunctionComponent } from "react";
 import { toast, ToastOptions } from "react-toastify";
 
 import { Alert, ToastType } from "~/components/alert";
 import { Icon } from "~/components/assets";
 import { t } from "~/hooks";
 
+export function dismissAllToasts() {
+  toast.dismiss();
```

Comment:
Why do we need a different function for that? Do you assume you will need to add more shared logic here in the future?